src/components/DocExplorer.js

```diff
@@ -56,7 +56,10 @@ export class DocExplorer extends React.Component {
   constructor() {
     super();
 
-    this.state = { navStack: [] };
+    this.state = {
+      navStack: [],
+      searchValue: null
```

Comment:
A related note about navStack for the future: it would be cool if scroll position were also maintained as part of the back stack. Today if you click from a long scrolling view to another long scrolling view your scroll position is maintained when it should probably be reset to "0", but then when going "back" the original scroll position should be restored. It's a fairly small detail, but it will help make navigating larger schema easier on the mind.